README.md

```diff
@@ -40,7 +40,15 @@ TODO
 
 Use with Up board
 -----------------
-TODO
+On the Up board, there is only a few points you need to check if you have issue with permissions (symptom is usually an error `Could not set necessary configuration of serial port` from the SBUS bridge)
```

Comment:
"there are"
"issues"
Can you link to the SBUS bridge since it is in a different repository?

Reply:
done